doc/api/modules.md

```diff
@@ -841,6 +841,14 @@ added: v9.3.0
 A list of  the names of all modules provided by Node.js. Can be used to verify
 if a module is maintained by a third-party module or not.
 
+Note that `module` in this context isn't the same object that's
+available by default inside file modules. To access it you need to
+require the `Module` module:
+
+```js
+const builtin = require('module').builtinModules;
+```
```

Comment:
That part is somewhat explained by the brief `Accessed via 'require("module")'` above.

Reply:
True, but to be honest I didn't see this my self. I needed to ask an adult for help. So I think it's worth showing an extra time. But maybe it's just me 😅